neo/Consensus/ConsensusService.cs

```diff
@@ -117,6 +117,10 @@ private void CheckCommits()
                 Block block = context.CreateBlock();
                 Log($"relay block: height={block.Index} hash={block.Hash} tx={block.Transactions.Length}");
                 localNode.Tell(new LocalNode.Relay { Inventory = block });
+
+                StateRoot stateRoot = context.CreateStateRoot();
+                Log($"relay stateRoot: height={stateRoot.Index} hash={stateRoot.Hash}");
```

Comment:
CreateStateRoot could return null, this should be checked here

Reply:
Fixed.